blog/fine-grained-news-2025-10.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Fine-Grained News - October 2025
+description: Fine-Grained News
+slug: fine-grained-news-2025-10
+date: 2025-10-15
+authors: aaguiar
+tags: [newsletter]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Fine-Grained News - October 2025
+
+Welcome to OpenFGA's Fine-Grained News, October edition!
+
+## 🎉 OpenFGA to CNCF Incubation!
+
+The CNCF completed the Due Diligence process to approve moving OpenFGA to the CNCF Incubation stage, and it's now [open for comments](https://github.com/cncf/toc/pull/1923). Support emojis are appreciated! 🚀 ❤️!
+
+Thanks to Grafana, Docker, Read.AI, Agicap, and Zuplo for taking the time to talk about their experience to the CNCF! 
+
+Moving to Incubation is great news for the OpenFGA community and we are super excited about it!
+
+## 🚀 What We've Shipped
+
+### OpenFGA v1.10.0 Performance Improvements
+
+Starting with [OpenFGA v1.10.0](https://github.com/openfga/openfga/releases/tag/v1.10.0), we've included a query planner that selects resolution strategies based on runtime statistics, behind the `enable-check-optimization` flag. The planner can pick different strategies per-relation, and we've seen significant improvements in performance for some models/relations.
+
+### Write Endpoint Enhancements  
+
+In case you missed it, it also adds additional parameters to the [`/write`](https://openfga.dev/api/service#/Relationship%20Tuples/Write) endpoint that allow specifying the behavior when duplicated tuples are written or non-existing tuples are deleted. We'll be completing SDK support for this feature in the next few weeks. 
```

Comment:
In case you missed it, we also included in [v1.10.0](https://github.com/openfga/openfga/releases/tag/v1.10.0) two new (optional) parameters to the [Write API](https://openfga.dev/api/service#/Relationship%20Tuples/Write) endpoint that allow specifying the expected behavior when duplicated tuples are written or non-existing tuples are deleted. We'll be completing SDK support (Go, Java, Python, .NET, JS, Ruby) for this feature in the next few weeks.